### DISCUSSION TITLE
How I can show a messager  icon for each item detail in select component, instead of just the text?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#5724] [Help]
Hi All,

How I can show a messager  icon for each item detail in select component, instead of just the text?

![image](https://user-images.githubusercontent.com/6214964/219856980-c86ecae9-c7db-405b-ba2f-5e2db40c4db8.png)

![image](https://user-images.githubusercontent.com/6214964/219856677-8baaa9de-efe2-43f4-96ef-739589802d82.png)

```
    Select::make('messager_app')->label('Messager')
        ->columnSpan(1)
        ->options([
            'whatsapp' => 'Whatsapp',
            'telegram' => 'Telegram',
            'line' => 'Line',
        ]),
```

Thanks

**Comment by ArtMin96** [comment]
[https://filamentphp.com/tricks/render-html-in-select-options](https://filamentphp.com/tricks/render-html-in-select-options)
